src/components/UnassignedColumn.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import Character from './Character.jsx'
+import { Droppable } from 'react-beautiful-dnd';
+
+// Single column of all the unassigned Characters, is droppable so characters can be returned to it
+const UnassignedColumn = ({column, characterList}) => {
```

Comment:
Could we combine `ActorColumn` and `UnassignedColumn` into one component since they're basically identical?   
`DropColumn`?